.github/workflows/flutter.yml

```diff
@@ -21,8 +21,53 @@ jobs:
       with:
         java-version: '12.x'
     - uses: subosito/flutter-action@v1.5.3
-    - run: flutter pub get
+
+    - name: Install dependencies for google_ml_kit
+      working-directory: ./packages/google_ml_kit
+      run: flutter pub get
+
+    - name: Install dependencies for google_ml_kit_barcode_scanning
+      working-directory: ./packages/google_ml_kit_barcode_scanning
+      run: flutter pub get
+
+    - name: Install dependencies for google_ml_kit_commons
+      working-directory: ./packages/google_ml_kit_commons
+      run: flutter pub get
+
+    - name: Install dependencies for google_ml_kit_face_detection
+      working-directory: ./packages/google_ml_kit_face_detection
+      run: flutter pub get
+
+    - name: Install dependencies for google_ml_kit_image_labeling
+      working-directory: ./packages/google_ml_kit_image_labeling
+      run: flutter pub get
+
+    - name: Install dependencies for google_ml_kit_digital_ink_recognition
+      working-directory: ./packages/google_ml_kit_digital_ink_recognition
+      run: flutter pub get
+
+    - name: Install dependencies for google_ml_kit_language_id
+      working-directory: ./packages/google_ml_kit_language_id
+      run: flutter pub get
+
+    - name: Install dependencies for google_ml_kit_object_detection
+      working-directory: ./packages/google_ml_kit_object_detection
+      run: flutter pub get
+
+    - name: Install dependencies for google_ml_kit_pose_detection
+      working-directory: ./packages/google_ml_kit_pose_detection
+      run: flutter pub get
+
+    - name: Install dependencies for google_ml_kit_text_recognition
+      working-directory: ./packages/google_ml_kit_text_recognition
+      run: flutter pub get
+
+    - name: Install dependencies for google_ml_kit_translation
+      working-directory: ./packages/google_ml_kit_translation
+      run: flutter pub get
+
```

Comment:
@bharat-biradar: the analyzer is failing,  you need to add the step for the new plugin here